templates/cloud/storage.html

```diff
@@ -10,7 +10,7 @@
       <div class="col-6">
         <h1>Ubuntu&nbsp;Advantage&nbsp;Storage</h1>
         <p>Ubuntu Advantage Storage from Canonical embeds the proven software&dash;defined storage (<abbr title="Software&dash;defined storage">SDS</abbr>) technologies of Ceph, NexentaEdge, Swift and SwiftStack, into a 24x7&dash;supported software solution with a unique metered pricing model.</p>
-        <p><a class="p-button--brand" href="/cloud/contact-us?product=storage">Request a demo</a></p>
+        <p><a class="p-button--brand" href="/cloud/contact-us?product=cloud-storage">Request a demo</a></p>
```

Comment:
I meant to update the form... and managed-cloud is now cloud-managed-cloud as well.